core/lib/zksync_core/src/state_keeper/io/common/mod.rs

```diff
@@ -107,3 +107,16 @@ pub(crate) async fn load_pending_batch(
         pending_l2_blocks,
     })
 }
+
+/// Clears pending l2 block data from the database.
+pub(crate) async fn clear_pending_l2_block(
```

Comment:
Bikeshedding: I'd place this as a static method in `L2BlockSealProcess`; seems to logically belong there.